packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -205,6 +205,35 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
         context.replaceVariables,
         options.timeZone
       );
+
+      if (field.config.custom?.nested) {
+        for (const nestedFrames of field.values) {
+          for (const nf of nestedFrames) {
+            for (const valueField of nf.fields) {
+              valueField.state = {
+                scopedVars: {
+                  __dataContext: {
+                    value: {
+                      data: options.data!,
+                      frame: nf,
+                      frameIndex: index,
+                      field: valueField,
```

Comment:
data, and frameIndex look incorect here (not the nested frame index, and data should be nestedFrames I think)

Reply:
ahh, thank you